container/os-autoinst_dev/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM opensuse/tumbleweed
 ENV NAME openQA test environment
 ENV LANG en_US.UTF-8
 
-RUN zypper ar -f -G 'http://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed' devel_openqa
+RUN zypper ar -f -G 'http://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed' devel_openqa && zypper -n in system-user-qemu && zypper clean
```

Comment:
Please add a comment here to explain the workaround

Reply:
done